cardano-node/src/Cardano/CLI/Ops.hs

```diff
@@ -421,17 +422,19 @@ localInitiatorNetworkApplication proxy cfg =
       (NodeToClientVersionData { networkMagic = nodeNetworkMagic proxy cfg })
       (DictVersion nodeToClientCodecCBORTerm) $ \_peerid ->
 
-      nodeToClientProtocols
-        (InitiatorProtocolOnly $
-           MuxPeer
-             nullTracer
-             localTxSubmissionCodec
-             (localTxSubmissionClientPeer localTxSubmissionClientNull))
-        (InitiatorProtocolOnly $
-           MuxPeer
-             nullTracer
-             localChainSyncCodec
-             (chainSyncClientPeer chainSyncClient))
+      nodeToClientProtocols $ NodeToClientProtocols
+      { localChainSyncProtocol = InitiatorProtocolOnly $
+                                   MuxPeer
+                                     nullTracer
+                                     localTxSubmissionCodec
+                                     (localTxSubmissionClientPeer localTxSubmissionClientNull)
+      , localTxSubmissionProtocol = InitiatorProtocolOnly $
+                                      MuxPeer
+                                        nullTracer
+                                        localChainSyncCodec
+                                        (chainSyncClientPeer chainSyncClient)
+      }
+
```

Comment:
These two are the wrong way around. Look at the names.

The fact that this does not cause a test failure shows that we do not have tests for the `getLocalTip` feature. Please file a ticket to add tests for this.

Reply:
Yep this slipped by me in the update. I hadn't tested that the node works as yet because I wanted to fix the windows build errors before testing the scripts.